src/components/number-input/stepper.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {ChevronDownIcon, ChevronUpIcon} from '../../icons';
+
+const styles = require('./number-input.st.css').default;
+
+function noop() {}
+
+export interface StepperProps extends React.HTMLProps<HTMLElement> {
+    onUp?: Function
+    onDown?: Function
+    disableUp?: boolean
+    disableDown?: boolean
+}
+
+export const Stepper: React.StatelessComponent<StepperProps> =
+    ({
+        onUp = noop,
+        onDown = noop,
+        disableUp = false,
+        disableDown = false,
+        ...props
+    }) => (
+        <div {...props}>
+            <button
+                tabIndex={-1}
+                data-automation-id="STEPPER_INCREMENT"
+                className={styles['stepper-control.stepper-increment']}
+                onClick={() => onUp()}
```

Comment:
Yes, because of the arguments. Any better suggestions? Could not come up with anything better here.